isis/src/base/apps/phocube/phocube.xml

```diff
@@ -627,10 +634,52 @@ xsi:noNamespaceSchemaLocation=
         <brief>Create an offNadir angle band</brief>
         <description>
           If this parameter is true, the <def>Off Nadir Angle</def> will be
-	  computed for every pixel and placed in a band in the output cube.
-	  The output cube labels will contain "OffNadir Angle" in the BandBin
-	  group, in band sequence of the output file.  OffNadirAngle is
-	  in degrees.
+          computed for every pixel and placed in a band in the output cube.
+          The output cube labels will contain "OffNadir Angle" in the BandBin
+          group, in band sequence of the output file.  OffNadirAngle is
+          in degrees.
+        </description>
+      </parameter>
+      <parameter name="SLOPE">
+        <type>boolean</type>
+        <default><item>FALSE</item></default>
+        <brief>Create a slope band</brief>
+        <description>
+          If this parameter is true, the slope will be computed for every pixel
+          and placed in a band in the output cube. The output cube labels will
+          contain "Slope" in the BandBin group, in band sequence of the output
+          file. The slope is in degrees with 0 degrees being perfectly flat and
+          90 degrees being sheer vertical.
+
+          If computing slopes, the input image must be a level1 image. To
+          compute slopes for a level2 DEM use
+          <a href="../slpmap/slpmap.html" target="_blank">slpmap</a>.
+        </description>
+      </parameter>
+      <parameter name="LOCALNORMAL">
+        <type>boolean</type>
+        <default><item>FALSE</item></default>
+        <brief>
+          Create X, Y, and Z back planes from the local normal vector
+        </brief>
+        <description>
+          If this parameter is true, the local normal will be computed for
+          every pixel and placed in three bands (X, Y, Z) in the output cube.
+          The output cube labels will contain three "Local Normal (coord)" keywords
+          in the BandBin group, in band sequence of the output file.
+        </description>
+      </parameter>
+      <parameter name="ELLIPSOIDNORMAL">
+        <type>boolean</type>
+        <default><item>FALSE</item></default>
+        <brief>
+          Create X, Y, and Z back planes from the ellipsoid normal vector
+        </brief>
+        <description>
+          If this parameter is true, the ellipsoid normal will be computed for
+          every pixel and placed in three bands (X, Y, Z) in the output cube.
+          The output cube labels will contain three "Ellipsoid Normal (coord)"
+          keywords in the BandBin group, in band sequence of the output file.
```

Comment:
Same reference frame comment as local normal, except indicate that this is computed from the triaxial ellipsoid.